Appreciation.AzureFunction/Function1.cs

```diff
@@ -0,0 +1,51 @@
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.WebJobs.Host;
+
+namespace Appreciation.Function
+{
+    public static class Function1
+    {
+        [FunctionName("Function1")]
+        public static async Task<HttpResponseMessage> Run([HttpTrigger(AuthorizationLevel.Function, "get", "post", Route = null)]HttpRequestMessage req, TraceWriter log)
```

Comment:
These two functions are just default? can we remove these?

Reply:
Yeah this should not have gone in there! I thought I'd deleted it before pushing, my bad.